<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input_dialog_frame</class>
 <widget class="QFrame" name="input_dialog_frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QLabel" name="order_nr_view_lbl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string>2000100000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="meta_text_edit">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>70</y>
     <width>1251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #AAFFFFFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="esc_button">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>690</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#FFFFFFFF</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>690</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#FFFFFFFF</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="formula_table">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>1251</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #AAFFFFFF</string>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>36</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>36</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="edit_item_group_box">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>971</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>edit selected item:</string>
   </property>
   <widget class="QPushButton" name="set_item_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/apply.png</normaloff>../images/apply.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantity_line_edit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pigment_lbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>pigment:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="pigment_combo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="quantity_lbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>quantity (gr):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="order_nr_lbl">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string>barcode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="warning_lbl">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>741</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_item_btn">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>580</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/add.png</normaloff>../images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="jars_to_add_lbl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>664</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/jar-green.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="n_of_jars_spinbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>654</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="jars_to_add_lbl_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>664</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/jar-green.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
